### PR TITLE
feat(integrations): add support for Fortnox

### PR DESCRIPTION
**Pull request by bodinsamuel**
## Changes

- Add support for Fortnox 
on behalf of a customer because there was an issue, but then it turns out there was no issue just something miss-configured.

### CODE REVIEW

File: packages/shared/providers.yaml [hunk 1]
```diff
@@ -2601,6 +2601,27 @@ fitbit:
...
     docs: https://docs.nango.dev/integrations/all/fitbit
 
+fortnox:
+    display_name: Fortnox
+    categories:
+        - accounting
+        - invoicing
+    auth_mode: OAUTH2
+    authorization_url: https://apps.fortnox.se/oauth-v1/auth
+    token_url: https://apps.fortnox.se/oauth-v1/token
```
Comment (nalanj) [inline packages/shared/providers.yaml:2611]: Pretty wild that their oauth2 flow is under a path named `oauth-v1`.

**Review by nalanj — APPROVED** [review]
:shipit:

File: packages/shared/providers.yaml (hunk 1 above)
Comment (bodinsamuel) [reply packages/shared/providers.yaml:2611]: yes it took me a while to stop debugging oauth v1 :D

Comment (TBonnin) [reply packages/shared/providers.yaml:2611]: 😮‍💨

**Review by TBonnin — APPROVED** [review]

**Pull request merged**